src/test/java/com/google/gcp_variant_transforms/options/VcfToBqContextTest.java

```diff
@@ -0,0 +1,127 @@
+// Copyright 2020 Google LLC
+ 
+package com.google.gcp_variant_transforms.options;
+ 
+import static com.google.common.truth.Truth.assertThat;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+ 
+import com.google.common.collect.ImmutableList;
+import java.io.IOException;
+import org.junit.Test;
+ 
+/**
+* Units tests for VcfToBqContext.java
+*/
+public class VcfToBqContextTest {
+  
+  private static final String INPUT_FILE = "sampleInputFile.txt";
+  private static final String OUTPUT = "sampleOutput";
+ 
+  public VcfToBqOptions create_mockedVcfToBqOptions(String inputFile, String output) {
+    VcfToBqOptions mockedVcfToBqOptions = mock(VcfToBqOptions.class);
+    when(mockedVcfToBqOptions.getInputFile()).thenReturn(inputFile);
+    when(mockedVcfToBqOptions.getOutput()).thenReturn(output);
+    return mockedVcfToBqOptions;
+  }
+ 
+  public ImmutableList<String> createHeaderLines() {
+    ImmutableList.Builder<String> headerLinesBuilder = new ImmutableList.Builder<String>();
+    for (int i = 0; i < 10; i++) {
+      headerLinesBuilder.add("##sampleHeaderLine=" + i);
+    }
+    return headerLinesBuilder.build();
+  }
+ 
+  @Test
+  public void testVcfContextConstructor_whenCompareFields_thenMatches() throws IOException {
+    VcfToBqOptions mockedVcfToBqOptions = create_mockedVcfToBqOptions(INPUT_FILE, OUTPUT);
+    VcfToBqContext vcfToBqContext = new VcfToBqContext(mockedVcfToBqOptions);
+ 
+    assertThat(vcfToBqContext.getInputFile()).matches(INPUT_FILE);
+    assertThat(vcfToBqContext.getOutput()).matches(OUTPUT);
+    assertThat(vcfToBqContext.getHeaderLines()).isNull();
+ }
+ 
+  @Test
+  public void testVcfContext_whenValidateFlags_thenException(){
+    VcfToBqOptions mockedVcfToBqOptions = create_mockedVcfToBqOptions(null, null);
+   
+    assertThrows(IOException.class, () ->
+        new VcfToBqContext(mockedVcfToBqOptions)); 
+  }
+ 
+  @Test
+  public void testVcfContext_whenGetInputFile_thenMatches() throws IOException {
+    VcfToBqOptions mockedVcfToBqOptions = create_mockedVcfToBqOptions(
```

Comment:
This object is used in many tests. I think we should create it is as the canonical options and use everywhere, while tests where "broken" options are used would still create their own.